tools/clang/test/CodeGenSPIRV/type.structured-buffer.vector.hlsl

```diff
@@ -0,0 +1,11 @@
+// Run: %dxc -T ps_6_0 -E main
```

Comment:
Please add 2 more tests identical to this test, but for one of them add `-fvk-use-dx-layout` and for the second one add `-fvk-use-scalar-layout`

Reply:
Done